celements-scheduler/src/main/java/com/celements/scheduler/classdefs/SchedulerJobClassPackage.java

```diff
@@ -0,0 +1,37 @@
+package com.celements.scheduler.classdefs;
+import java.util.List;
+
+import javax.inject.Inject;
+import javax.validation.constraints.NotEmpty;
+import javax.validation.constraints.NotNull;
+
+import org.springframework.beans.factory.ListableBeanFactory;
+import org.springframework.stereotype.Component;
+
+import com.celements.model.classes.AbstractClassPackage;
+import com.celements.model.classes.ClassDefinition;
+import com.celements.scheduler.classdefs.SchedulerJobClassDefinition;
+
+@Component
+public class SchedulerJobClassPackage extends AbstractClassPackage {
```

Comment:
please extend `AbstractLegacyClassPackage` to support existing XWikiPreferences configuration with the field 'activated_classcollections'